schemas/JSON/manifests/v1.2.0/manifest.locale.1.2.0.json

```
@@ -149,6 +164,23 @@
       "$ref": "#/definitions/Url",
       "description": "The package release notes url"
     },
+    "PurchaseUrl": {
+      "$ref": "#/definitions/Url",
+      "description": "The purchase url for acquiring entitlement for the package."
+    },
+    "InstallationNotes": {
+      "type": [ "string", "null" ],
+      "minLength": 1,
+      "maxLength": 240,
+      "description": "The notes displayed to the user upon completion of a package installation."      
+    },
+    "Documentations": {
+      "type": [ "array", "null" ],
+      "items": {
+        "$ref": "#/definitions/Documentation"
+      },
+      "maxItems": 256

```

If documentations exists, then it should not be an empty array
```suggestion
      "minItems":1,
      "maxItems": 256
```